web/locales/en/plugin__network-observability-plugin.json

```diff
@@ -19,23 +20,23 @@
   "Clear all filters and try again.": "Clear all filters and try again.",
   "Column management": "Column management",
   "Refresh off": "Refresh off",
-  "{{count}} second": "{{count}} second",
-  "{{count}} second_plural": "{{count}} seconds",
-  "{{count}} minute": "{{count}} minute",
-  "{{count}} minute_plural": "{{count}} minutes",
-  "{{count}} hour": "{{count}} hour",
-  "{{count}} hour_plural": "{{count}} hours",
-  "{{count}} day": "{{count}} day",
-  "{{count}} day_plural": "{{count}} days",
+  "{{count}} second_one": "{{count}} second",
+  "{{count}} second_other": "{{count}} second",
+  "{{count}} minute_one": "{{count}} minute",
+  "{{count}} minute_other": "{{count}} minute",
+  "{{count}} hour_one": "{{count}} hour",
+  "{{count}} hour_other": "{{count}} hour",
+  "{{count}} day_one": "{{count}} day",
+  "{{count}} day_other": "{{count}} day",
   "Custom time range": "Custom time range",
-  "Last {{count}} minute": "Last {{count}} minute",
-  "Last {{count}} minute_plural": "Last {{count}} minute",
-  "Last {{count}} hour": "Last {{count}} hour",
-  "Last {{count}} hour_plural": "Last {{count}} hour",
-  "Last {{count}} day": "Last {{count}} day",
-  "Last {{count}} day_plural": "Last {{count}} day",
-  "Last {{count}} week": "Last {{count}} week",
-  "Last {{count}} week_plural": "Last {{count}} week",
+  "Last {{count}} minute_one": "Last {{count}} minute",
+  "Last {{count}} minute_other": "Last {{count}} minute",
+  "Last {{count}} hour_one": "Last {{count}} hour",
+  "Last {{count}} hour_other": "Last {{count}} hour",
+  "Last {{count}} day_one": "Last {{count}} day",
+  "Last {{count}} day_other": "Last {{count}} day",
+  "Last {{count}} week_one": "Last {{count}} week",
+  "Last {{count}} week_other": "Last {{count}} week",
```

Comment:
Did we missed something in previous PRs ? 

Looks like we are loosing all plurals

Reply:
Yesss that's very strange. I double-checked this and it is consistent with what it's in the main branch right now:

https://github.com/netobserv/network-observability-console-plugin/blob/main/web/src/components/time-range-dropdown.tsx#L32-L43